src/api/app/views/webui/request/_action_text.html.haml

```diff
@@ -0,0 +1,10 @@
+- if (details = action.commit_details)
+  %p.mt-0.mb-0.font-weight-normal
+    = details['comment']&.truncate(50) || action.name
+    %span.float-right.ml-5
+      = details['rev']
```

Comment:
![Screenshot 2022-09-12 at 10-55-44 Open Build Service](https://user-images.githubusercontent.com/22001671/189613533-d48de1a1-26d7-4a76-a633-38881fcb103f.png)

To me its not clear on the first glance what the revision number means. I just saw the number 2 and started thinking what it possibly means. But could be a follow up to somehow better indicate what this number is about

Reply:
It didn't take too long, so we decided to improve this a bit. This is how it looks like now:

![Screenshot 2022-09-12 at 11-23-20 Open Build Service](https://user-images.githubusercontent.com/2581944/189619932-ce607269-9b19-45df-bf57-238096460ee5.png)

Reply:
Cool, much better